src/command_modules/azure-cli-backup/azure/cli/command_modules/backup/__init__.py

```diff
@@ -0,0 +1,14 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import azure.cli.command_modules.backup._help #pylint: disable=unused-import
+
+
+def load_params(_):
+    import azure.cli.command_modules.backup._params #pylint: disable=redefined-outer-name
```

Comment:
Format:
1. Two spaces between `#` and the preceding character.
2. One space between `#` and the following character.